activerecord/lib/active_record/railtie.rb

```diff
@@ -26,6 +26,7 @@ class Railtie < Rails::Railtie # :nodoc:
 
     config.active_record.use_schema_cache_dump = true
     config.active_record.maintain_test_schema = true
+    config.active_record.belongs_to_required_by_default = true
```

Comment:
@rafaelfranca I thought these are initial values, so initializers can use the default configuration and change them. Anyway, thank you for your review.